From be79e6a2a8c99d96122e5f568028b8963786febb Mon Sep 17 00:00:00 2001
From: "guoyin.chen" <guoyin.chen@freescale.com>
Date: Tue, 30 Jul 2013 14:03:57 +0800
Subject: [PATCH 224/304] ENGR00273070 Enable 3G data connection for JB4.3

Align the version number as in frameworks/base/core/res/res/xml/apns.xml

Signed-off-by: guoyin.chen <guoyin.chen@freescale.com>
---
 imx5x/apns-conf.xml    | 2 +-
 imx6/etc/apns-conf.xml | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/imx5x/apns-conf.xml b/imx5x/apns-conf.xml
index 6a7f655..ab8362f 100644
--- a/imx5x/apns-conf.xml
+++ b/imx5x/apns-conf.xml
@@ -19,7 +19,7 @@
 
 <!-- use empty string to specify no proxy or port -->
 <!-- This version must agree with that in apps/common/res/apns.xml -->
-<apns version="7">
+<apns version="8">
     <apn carrier="T-Mobile US"
          mcc="310"
          mnc="260"
diff --git a/imx6/etc/apns-conf.xml b/imx6/etc/apns-conf.xml
index 6a7f655..ab8362f 100644
--- a/imx6/etc/apns-conf.xml
+++ b/imx6/etc/apns-conf.xml
@@ -19,7 +19,7 @@
 
 <!-- use empty string to specify no proxy or port -->
 <!-- This version must agree with that in apps/common/res/apns.xml -->
-<apns version="7">
+<apns version="8">
     <apn carrier="T-Mobile US"
          mcc="310"
          mnc="260"
-- 
1.8.0

